### DISCUSSION TITLE
error update  packages  orchardcore of 1.72 to 1.8.2

### DISCUSSION THREAD

**Discussion by ODA669** [OrchardCMS/OrchardCore#15720] [General]
Good morning, someone has a problem trying to update Orchard modules to 1.8.2. I tried upgrading from 1.7.2 to 1.8.2 and this broke my project after upgrading
all references i have  no matter which update the project breaks

    <PackageReference Include="OrchardCore.Application.Cms.Core.Targets" Version="1.7.2" />
    <PackageReference Include="OrchardCore.Title" Version="1.7.2" />
    <PackageReference Include="OrchardCore.Translations.All" Version="1.7.1" />
  <PackageReference Include="OrchardCore.Abstractions" Version="1.7.2" />
  <PackageReference Include="OrchardCore.ContentLocalization.Abstractions" Version="1.7.2" />
  <PackageReference Include="OrchardCore.ContentManagement.Display" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Contents.TagHelpers" Version="1.7.2" />
  <PackageReference Include="OrchardCore.CustomSettings" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Data" Version="1.7.2" />
  <PackageReference Include="OrchardCore.DisplayManagement" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Localization" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Localization.Abstractions" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Localization.Core" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Module.Targets" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Navigation" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Rules.Abstractions" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Settings.Core" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Setup" Version="1.7.2" />
  <PackageReference Include="OrchardCore.Theme.Targets" Version="1.7.2" />
  <PackageReference Include="OrchardCore.ContentManagement" Version="1.7.2" />
  <PackageReference Include="OrchardCore.ResourceManagement" Version="1.7.2" />

 [orchardcore.application.cms.core.target](http://orchardcore.application.cms.core.target/) and orchardcore.abstraction. this error : An unhandled exception occurred while processing the request.
MissingMethodException: Method not found: 'Void [OrchardCore.Environment.Shell](http://orchardcore.environment.shell/).Configuration.ShellConfiguration..ctor(System.String, System.Func2<System.String,System.Threading.Tasks.Task1<Microsoft.Extensions.Configuration.IConfigurationBuilder>>)'.
[OrchardCore.Environment.Shell](http://orchardcore.environment.shell/).ShellSettingsManager.LoadSettingsAsync(string tenant)

**Comment by Piedone** [comment]
This is the same issue as @effapp is having here: https://github.com/OrchardCMS/OrchardCore/issues/15642, please chime in there.
